bal_tools/graphql/blocks/first_block_after_ts.gql

```diff
@@ -1,9 +1,9 @@
-query FirstBlockAfterTimestamp($timestamp: BigInt) {
+query FirstBlockAfterTimestamp($timestamp_gt: BigInt, $timestamp_lt: BigInt) {
   blocks(
     first: 1
     orderBy: number
     orderDirection: asc
-    where: { timestamp_gt: $timestamp }
+    where: { timestamp_gt: $timestamp_gt, timestamp_lt: $timestamp_lt}
```

Comment:
Are you sure nothing is using this yet and passing only GT?

Reply:
not sure where else its used outside of this/fee allocator currently. this query is called from `get_first_block_after_utc_timestamp`, so should be implied that its gt